packages/dd-trace/src/appsec/rasp/utils.js

```diff
@@ -46,7 +46,8 @@ function handleResult (actions, req, res, abortController, config) {
   if (blockingAction) {
     const rootSpan = web.root(req)
     // Should block only in express
-    if (rootSpan?.context()._name === 'express.request') {
+    const name = rootSpan?.context()._name
+    if (name === 'express.request' || name === 'web.request') {
```

Comment:
Question for the APM team: Why do `Express5` spans have a `web` prefix instead of an `express` prefix?

Reply:
Not related after route and use instrumentation